src/briefcase-using.rst

```diff
@@ -0,0 +1,231 @@
+Using ODK Briefcase
+======================
+
+.. _pull-from-aggregate:
+
+Pulling forms from Aggregate
+-----------------------------
+
+To download blank forms and completed form instances from an :doc:`Aggregate <aggregate-intro>` server:
+
+#. Open the :guilabel:`Pull` tab.
+#. Select *Aggregate 1.0* in the :guilabel:`Pull data from` drop-down.
+#. Click the :guilabel:`Connect` button and enter the URL and login credentials for your Aggregate server.
+
+   If you have anonymous login enabled on Aggregate, no login credentials are needed here.
+   
+   To connect to the `Aggregate Demo Server`_, the URL is https://opendatakit.appspot.com.
+   
+   .. _Aggregate Demo Server: https://opendatakit.appspot.com
+
+#. Select the forms you want to download and click :guilabel:`Pull`. The selected forms will be pulled to :file:`ODK Briefcase Storage` on your local system.
+
+   For each selected form, Briefcase will pull down:
+   
+   - The form definition file (that is, the blank XForm).
+   - All media associated with the form.
+   - Completed form instances.
+
+   If you have previously pulled the form:
+   
+   - New instances will be downloaded to the same location as previously downloaded ones.
+   - The form definition file and media files will not be re-copied.
+   
+   .. warning::
```

Comment:
In the rendered page, the warning looks is too close to the previous text. Maybe some CSS glitch? It's minor. 

<img width="751" alt="screen shot 2018-03-08 at 11 28 46" src="https://user-images.githubusercontent.com/32369/37171801-e28ce92c-22c3-11e8-9fdf-53520e550cf1.png">

Reply:
This is related to #538